src/main/resources/test_bqQuery/testSuite.json

```diff
@@ -0,0 +1,10 @@
+{
+  "caseAutoDetect": true,
+  "executions":[
+    {
+        "contextType": "BqQuery",
```

Comment:
it should be called executionType
and you need to specify here the context where the query will be executed. You can but it in a  "system" property